samples/azure-quantum/variational-quantum-eigensolver/README.md

```diff
@@ -0,0 +1,20 @@
+---
+page_type: sample
+author: guenp
+description: Variational Quantum Eigensolver
+ms.author: guenp@microsoft.com
+ms.date: 05/02/2022
+languages:
+- python
+products:
+- azure-quantum
+---
+
+# Variational Quantum Eigensolver
+
+This is an Azure Quantum sample notebook that illustrates how to implement and run a Variational Quantum Eigensolver (VQE) program.
```

Comment:
Especially since this will be discoverable from the Microsoft-wide samples gallery and since we have several other VQE samples, we should have more description here and in the title to help users correctly place this sample into context.

Reply:
Would you be able to update the README with more details about the VQE sample? I think Guen worked on this sample and merged this README, so I don't have much context here.

Reply:
Yes, absolutely; I won't have a chance for a few hours, but will get to it ASAP.